### DISCUSSION TITLE
Card as dropdown cutting on drawer-content

### DISCUSSION THREAD

**Discussion by ravisinghnath** [saadeghi/daisyui#2145] [Q&A] [answered]
I am using responsive drawer component with card as dropdown component inside drawer-side but the card popup cuts with drawer-content. I will appreciate any help.

https://play.tailwindcss.com/UF4SAEdpK0?layout=preview

**Comment by saadeghi** [comment, marked as answer]
You can fix this by adding `overflow-visible` to `drawer-side`  
But in cost of losing the vertical scroll if the sidebar menu is long enough to scroll.  

The reason is: in CSS when we make an element scrollable in `y` axis, the `x` axis can't have an overflow like default.
